### DISCUSSION TITLE
How can I set ignoreUndefinedProperties?

### DISCUSSION THREAD

**Discussion by renep16** [gladly-team/next-firebase-auth#462] [Q&A]
Hi, I have a question about setup and how can I set ignoreUndefinedProperties? For example I added this on a API in Next
const db = getFirebaseAdmin().firestore()
db.settings({ignoreUndefinedProperties: true})

But only worked once, then I got the error:
Error: Firestore has already been initialized. You can only call settings() once, and only before calling any other methods on a Firestore object.
    at Firestore.settings
    
  As understand in your project you have the init function to initialize the firebase app if is not, but i can not give settings there,

**Comment by kmjennison** [comment]
Thanks for the question. See #191—others have had this issue, too, and we need a simple reproduction to see if we can help.

**Reply by renep16** [reply]
I have checked the example before but not sure if it also init and set the firebase admin. Since it is using firebase.initializeApp() with the firebaseClientInitConfig and dont see something about the admin. In my case I want to use the ignoreUndefinedProperties more for admin side
